### ISSUE TITLE
Problem tracking media_position accurately of media_player in slider

### ISSUE THREAD

**Issue by iamthekings5** [Nerwyn/service-call-tile-feature#69]
**Checklist:**

- [/] I updated to the latest version available
- [/] I cleared the cache of my browser
- [/] I have confirmed that this is an issue with this project and not an integration by making the same actions with the [actions dev tool](http://homeassistant.local:8123/developer-tools/action)

**Release with the issue:**
4.2.2

**Last working release (if known):**
N/A

**Browser and Operating System:**

Google Chrome with Home Assistant as PWA, Windows 11 Home 23H2

**Description of problem:**

I am attempting to build a media player out of a tile card using this custom integration. I have copied most of my settings for the slider representing the media_position from your example in the read me.
![image](https://github.com/user-attachments/assets/81472306-e7dd-4c65-abbc-533c4bf689eb)
![image](https://github.com/user-attachments/assets/ab30847d-8d83-43e0-9aa5-0a3abcc57528)

Notice how when the media player is paused and playing, the media position is not the same despite the fact that they are taken within 1s of each other. When I click play, the media position jumps back (by 30, will explain more), and when I click pause, it jumps forward.

When inspecting, I notice the value jumps back and forward by 30. When it is at the start of the song, when the song is playing the value can report negative numbers, though not offset by exactly 30. I am unable to isolate the logic behind how the values are offset exactly, but at some points it is offset by consistently 30.

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Your Partial Card Config**

```yaml
  - type: custom:service-call
    entries:
      - type: slider
        entity_id: media_player.spotify
        range:
          - 0
          - "{{ state_attr(config.entity, 'media_duration') }}"
        tap_action:
          action: perform-action
          target:
            entity_id: media_player.spotify
          perform_action: media_player.media_seek
          data:
            seek_position: "{{ value }}"
        step: 1
        thumb: flat
        value_attribute: media_position
        value_from_hass_delay: 5000
        styles: |-
          :host {
            --color: var(--primary-color);
            flex-basis: 475%;
            height: 12px;
            --thumb-width: 0px;
          }
```

**Additional information:**
I did not post my full config as it is extremely long and I currently do not have time to scrub it of personal data, but if you believe it is vital to understanding the issue, let me know and I will post the full config.

I have tried varying value_from_hass_delay from 0, 1000, 5000 and 50000 but the issue persists.

I thought the cause could have been from either value_attribute: media_position or the end value of range:. Inspecting shows that the range remains constant throughout the song whether paused or playing, but the value is offset when pausing and playing, isolating the issue. Checking the state attribute of media_player.spotify shows no patterns of the value being changed, suggesting the problem is in how value_attribute: media_position is being passed.

I am using the SpotifyPlus custom integration, not the default one.

**Comment by Nerwyn** [comment]
[Logic for setting value when attribute is `media_position` for reference](https://github.com/Nerwyn/service-call-tile-feature/blob/4.2.2/src/classes/base-custom-feature.ts#L388-L429).

When the media player's state isn't `playing`, it just uses the `media_position` attribute value returned by Home Assistant, so I don't see why the value would be incorrect while paused. Are you sure this isn't a bug with SpotifyPlus returning the wrong `media_position` when paused? Or returning incorrect `media_position`, `media_duration`, and/or `media_position_updated_at` attributes when playing? I'm not able to recreate this issue with the default Home Assistant Spotify integration.

**Comment by Nerwyn** [comment]
I installed the SpotifyPlus integration and wasn't able to recreate the issue when using it's media player entity either.

**Comment by iamthekings5** [comment]
I have tested the issue a bit and it seems the issue only occurs on the following web browsers:
Firefox Desktop (Windows)
Chrome Desktop (Windows)
Edge Desktop (Windows)

But is not occurring on:
HA Companion App (Android)

The desktop browsers' behaviour are accurate when the player is paused, but when playing it jumps back a significant amount.

**I have also realised the issue is occurring in mini-media-player which I have installed suggesting the issue is not isolated to Custom Features for Home Assistant Cards.**

However, I am a bit confused as to what is causing this. When I inspect the attribute in developer tools I don't see skipping of any sort of behaviour. `media_position`, `media_duration`, and / or `media_position_updated_at` seem to be reporting accurate values.

I will test further in my free time, but if you have any idea as to what could be causing this, do let me know. Thank you for your time.
